pkg/syslLogger/testutil/testutil.go

```diff
@@ -0,0 +1,181 @@
+package testutil
```

Comment:
If I put it in the `loggers` package, it will expose the `testutil` functions to users. Since the main API and the `loggers` package use it, I thought it would be neater to put it in a separate package.

Reply:
That's cool. I wasn't sure whether it was used by one package or more.